src/Dfe.PlanTech.Web/Controllers/QuestionsController.cs

```diff
@@ -25,20 +32,30 @@ public async Task<IActionResult> GetQuestionById(string? id, string? section, [F
     {
         var parameterQuestionPage = TempData[TempDataConstants.Questions] != null ? DeserialiseParameter<TempDataQuestions>(TempData[TempDataConstants.Questions]) : new TempDataQuestions();
 
-        if (string.IsNullOrEmpty(id)) id = parameterQuestionPage.QuestionRef;
+        if (string.IsNullOrEmpty(id))
```

Comment:
In the current implementation, `id` will always be null. Can we change the code to reflect how it actually works/its purpose please? It is not very clear when/why each bit of logic would be executed without running through it.

Reply:
i've removed the id as a param as it's never used.